modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/SW360MetaDataUpdater.java

```diff
@@ -120,11 +110,11 @@ public void setUpdateReleases(boolean updateReleases) {
     }
 
     public void createProject(String projectName, String projectVersion, Collection<SW360Release> releases) throws AntennaException, IOException {
-        String id;
-        HttpHeaders header = createHttpHeaders(userId, password);
+        HttpHeaders header = sw360ConnectionConfiguration.getHttpHeaders();
 
         Optional<String> projectId = projectClientAdapter.getProjectIdByNameAndVersion(projectName, projectVersion, header);
 
+        String id;
         if (projectId.isPresent()) {
             // TODO: Needs endpoint on sw360 to update project on sw360
```

Comment:
is there an issue for this TODO?

Reply:
I do not know, but this TODO is also not added by me